CMakeLists.txt

```diff
@@ -105,6 +105,8 @@ elseif(BASEARCH_S360_FOUND)
     option(WITH_DFLTCC_INFLATE "Build with DFLTCC intrinsics for decompression on IBM Z" OFF)
     option(WITH_CRC32_VX "Build with vectorized CRC32 on IBM Z" ON)
 elseif(BASEARCH_X86_FOUND)
+    option(WITH_AVX512_VNNI "Build with AVX512 VNNI extensions" ON)
+    option(WITH_AVX512 "Build with AVX512" ON)
```

Comment:
Move to below AVX2

Reply:
.. and AVX512 should always go before AVX512VNNI imo.